src/index.js

```diff
@@ -0,0 +1,193 @@
+"use strict";
+
+
+const state = {
+    temp : 70, 
+    sky : document.getElementById('skySelect'),
+    location : '',
+    lon : 0,
+    lat : 0
+}
+
+// Elements Selectors 
+const tempValue = document.getElementById('tempValue');
+
+const actualTempValue = document.getElementById('actualValue')
+
+const increaseTempControl = document.getElementById('increaseTempControl');
+
+const decreaseTempControl = document.getElementById('decreaseTempControl');
+
+const skySelect = document.getElementById('skySelect');
+
+const desiredSky = document.getElementById('desiredSky')
+
+const desiredLandscape = document.getElementById('landscape')
+
+const bgColor = document.getElementById('backgroundColor')
+
+const headerCityName = document.getElementById('headerCityName')
+
+const cityNameInput = document.getElementById('cityNameInput')
+
+const submitButton = document.getElementById('cityNameSubmit')
+
+const resetButton = document.getElementById('cityNameReset')
+
+
+
+// Functions 
+// TEMP CHANGES
+const increaseTemp = () => {
+    state.temp ++;
+    tempValue.innerHTML = state.temp + "°" ;
+} 
+
+const decreaseTemp = () => {
+    state.temp --;
+    tempValue.innerHTML = state.temp + "°" ;
+}
+
+// CITY HEADER
+const changeCityName = () => {
+    headerCityName.innerHTML = cityNameInput.value 
+
+}
+
+// State.Location Update
+const locationUpdate = () => {
+    state.location = cityNameInput.value; 
+    getLocation();
+}
+
+const resetLocation = () => {
+    state.location = '';
+    cityNameInput.value = state.location;
+    changeCityName();
+}
```

Comment:
When you reset the city it should be updated to a selected city instead of an empty string like so:
```
  cityNameInput.value = 'Seattle';
```